time event

float32 x
float32 x_vel
float32 x_acc

float32 x_goal
float32 x_vel_goal
float32 x_acc_goal

float32 y
float32 y_vel
float32 y_acc

float32 y_goal
float32 y_vel_goal
float32 y_acc_goal

float32 z
float32 z_vel
float32 z_acc

float32 z_goal
float32 z_vel_goal
float32 z_acc_goal

float32 yaw
float32 yaw_vel
float32 yaw_acc

float32 yaw_goal
float32 yaw_vel_goal
float32 yaw_acc_goal
